src/TableColumn/Labels.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace atk4\ui\TableColumn;
+
+/**
+ * Class Labels
+ *
+ * take the fieldValue separated by commas and transforms into SemanticUI labels
+ *
+ * from => label1,label2 | to => div.ui.label[label1] div.ui.label[label2]
+ *
+ * @package atk4\ui\TableColumn
+ */
+class Labels extends Generic
+{
+    public function getHTMLTags($row, $field)
+    {
+        $values = explode(',', $field->get());
```

Comment:
Theoretically $field->get() can already return array if model field type=array, so I would wrap this in `if (is_string($field->get()))` like this

```
$values = $field->get();
if (is_string($values)) {
    $values = explode(',', $values);
}
```